### PR TITLE
media-gfx/maim version bump

### PR DESCRIPTION
**Pull request by mrVanDalo**

### CODE REVIEW

File: media-gfx/maim/maim-5.4.63.ebuild [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+DESCRIPTION="Commandline tool to take screenshots of the desktop"
+HOMEPAGE="https://github.com/naelstrof/maim"
+SRC_URI="https://github.com/naelstrof/maim/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND="${RDEPEND}
```
Comment (SoapGentoo) [inline media-gfx/maim/maim-5.4.63.ebuild]: this doesn't make a lot of sense, `RDEPEND` hasn't been defined yet

File: media-gfx/maim/maim-5.4.63.ebuild [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	x11-libs/libXcomposite
+	virtual/opengl
+	x11-misc/slop
+"
+
+RDEPEND="
+	media-libs/libjpeg-turbo
+	media-libs/libpng
```
Comment (SoapGentoo) [inline media-gfx/maim/maim-5.4.63.ebuild]: usually libpng needs `:0=`

**Comment by mrVanDalo** [comment]
Fixed your issues @SoapGentoo

**Comment by mrVanDalo** [comment]
@SoapGentoo ping

**Pull request closed without merging**